requirements/default.txt

```diff
@@ -179,22 +176,9 @@ pytest==3.2.0 \
 pytest-cov==2.5.1 \
     --hash=sha256:03aa752cf11db41d281ea1d807d954c4eda35cfa1b21d6971966cc041bbf6e2d \
     --hash=sha256:890fe5565400902b0c78b5357004aab1c814115894f4f21370e2433256a3eeec
-pytz==2017.2 \
-    --hash=sha256:03c9962afe00e503e2d96abab4e8998a0f84d4230fa57afe1e0528473698cdd9 \
-    --hash=sha256:39504670abb5dae77f56f8eb63823937ce727d7cdd0088e6909e6dcac0f89043 \
-    --hash=sha256:43f52d4c6a0be301d53ebd867de05e2926c35728b3260157d274635a0a947f1c \
-    --hash=sha256:487e7d50710661116325747a9cd1744d3323f8e49748e287bc9e659060ec6bf9 \
-    --hash=sha256:54a935085f7bf101f86b2aff75bd9672b435f51c3339db2ff616e66845f2b8f9 \
-    --hash=sha256:c883c2d6670042c7bc1688645cac73dd2b03193d1f7a6847b6154e96890be06d \
-    --hash=sha256:d1d6729c85acea5423671382868627129432fba9a89ecbb248d8d1c7a9f01c67 \
-    --hash=sha256:ddc93b6d41cfb81266a27d23a79e13805d4a5521032b512643af8729041a81b4 \
-    --hash=sha256:f5c056e8f62d45ba8215e5cb8f50dfccb198b4b9fbea8500674f3443e4689589
 raven==6.1.0 \
     --hash=sha256:02cabffb173b99d860a95d4908e8b1864aad1b8452146e13fd7e212aa576a884 \
     --hash=sha256:56dc9062dd42bca97350e5048ff417c914376366caa3b1b5f788b27ddc0a34b7
-redis==2.10.5 \
-    --hash=sha256:5dfbae6acfc54edf0a7a415b99e0b21c0a3c27a7f787b292eea727b1facc5533 \
-    --hash=sha256:97156b37d7cda4e7d8658be1148c983984e1a975090ba458cc7e244025191dbd
```

Comment:
There's a bunch of code that imports `redis`. Looks like it's used for caching and then a bunch of modules import it to get `RedisError`. Seems like we need to keep this in the `default.txt` file.

I don't see anything that uses hiredis. Doing a `pipdeptree` suggests it's not required by other packages.

Reply:
It's optional, used by the redis module if available. The explicit extra was added in redis 3.0.0. See https://github.com/mozilla/ichnaea/pull/825#discussion_r310264092.

Reply:
and 👍 to moving redis back. I confirmed that ``celery[redis]`` brings it in, but we can be more explicit.

Reply:
Ahh... ok--that makes sense. I didn't understand https://github.com/mozilla/ichnaea/pull/825#discussion_r310264092 .